src/resolver.cpp

```diff
@@ -1549,6 +1563,25 @@ namespace aDNS
       fmt::format("no suitable zone found for {}", std::string(name)));
   }
 
+  namespace
+  {
+    // TODO needs cleanup and wrappers to avoid memleaks (in CCF?..)
+    std::string get_common_name(const X509_NAME* name)
```

Comment:
TBC, but that sounds like functionality that's only needed for CA purposes, and is out of the scope for the PoC (DNS/DNSSEC/DANE/Attestation only), in which there should be no need to parse out CNs.